frontend/src/components/not-found.ts

```diff
@@ -9,27 +9,45 @@ import { BtrixElement } from "@/classes/BtrixElement";
 export class NotFound extends BtrixElement {
   render() {
     return html`
-      <div class="text-center text-neutral-500">
+      <div class="text-center">
         <p class="my-4 border-b py-4 text-xl leading-none text-neutral-400">
-          ${msg("Page not found")}
+          ${msg("Sorry, we couldn’t find that page")}
         </p>
-        <p>
+        <p class="text-neutral-600">
+          ${msg("Check the URL to make sure you’ve entered it correctly.")}
+        </p>
+        <div class="my-4">
+          <sl-button
```

Comment:
This doesn't really follow the style of the other buttons in the app, can we use a `<btrix-button>` instead?